### PR TITLE
Refactor push processor args

### PR DESCRIPTION
**Pull request by wfxr**

### CODE REVIEW

File: pkg/oomstore/types/config.go [hunk 1]
```diff
@@ -27,14 +27,13 @@ type OomStoreConfig struct {
...
 }
 
-type StreamPushProcessorConfig struct {
-	RevisionInterval time.Duration `yaml:"revision-interval"`
-	Period           time.Duration `yaml:"period"`
-	MinPeriod        time.Duration `yaml:"min-period"`
+type PushProcessorConfig struct {
 	BufferSize       int           `yaml:"buffer-size"`
+	FlushInterval    time.Duration `yaml:"flush-interval"`
+	RevisionInterval time.Duration `yaml:"revision-interval"`
```
Comment (jinghancc) [inline pkg/oomstore/types/config.go:36]: what's the difference between `FlushInterval` and `RevisionInterval`?

Comment (jinghancc) [reply pkg/oomstore/types/config.go:36]: btw, I remember we agree on `SnapshotInterval`?

Comment (wfxr) [reply pkg/oomstore/types/config.go:36]: `RevisionInterval` will be remove once @lianxmfor added `SnapshotInterval` in group metadata.
`FlushInterval` literally means interval between flushes.

File: pkg/oomstore/push_processor.go [hunk 2]
```diff
@@ -78,7 +85,7 @@ func (s *OomStore) InitStreamPushProcessor(ctx context.Context, cfg *types.Strea
 				processor.buffer.Range(func(key, value interface{}) bool {
 					groupID := key.(int)
 					b := value.(GroupBuffer)
-					if len(b.records) > 0 && time.Since(b.lastPush) > processor.minPeriod {
+					if len(b.records) > 0 && time.Since(b.lastPush) >= processor.flushInterval {
```
Comment (jinghancc) [inline pkg/oomstore/push_processor.go:88]: should we use the `min(maxTickInterval, processor.flushInterval)`

Comment (wfxr) [reply pkg/oomstore/push_processor.go:88]: explained offline.

**Pull request merged**